pkg/R/context.R

```diff
@@ -19,6 +19,10 @@
 #'}
 
 textFile <- function(sc, path, minSplits = NULL) {
+  # Allow the user to have a more flexible definiton of the text file path
+  path <- path.expand(path)
+  if (!(grepl("\\\\", path) | grepl("/", path)))
```

Comment:
@dputler - No worries about the git client. I can fix things up during merge. One last thing I wanted to check with you was whether using `normalizePath` is better than using `path.expand`. From the documentation it looks like `normalizePath` calls `path.expand` first and then also handles things like `../` etc.